inst/markdown/compareScenarios2/cs2_main.Rmd

```diff
@@ -207,11 +208,11 @@ rm(dataScen, dataScenNested, dataHist)
 data %>%
   # In the variable names, replace `$dot$` by `.`.
   mutate(
-    variable = remind2::substituePlaceholder(variable)) %>%
+    variable = substituePlaceholder(variable)) %>%
   # In the variable names, replace `|+|`, `|++|`, `|+++|`, ... by `|`.
   mutate(
     varplus = as.character(variable),
-    variable = remind2::deletePlus(variable)) ->
+    variable = deletePlus(variable)) ->
```

Comment:
Here's another `->` which should be replaced

Reply:
The cs2-Rmd-files do not really follow the code style conventions yet. There is much more to change there. I will do that in a different PR